src/views/Stake/Stake.tsx

```diff
@@ -16,7 +17,8 @@ const Stake: React.FC = () => {
   return (
     <Box display="flex" alignItems="center" justifyContent="center" flexDirection="column">
       <StakeArea />
-      <ExternalStakePools />
+      {/* <ExternalStakePools /> */}
```

Comment:
did you want to completely turn off the farm pools?

Reply:
yea that was in Tutti's design...